cvs diff -u -wb -i bakefiles\common.bkl bakefiles\config.bkl (in directory C:\wx\wxWindows\build\)
Index: bakefiles/common.bkl
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/build/bakefiles/common.bkl,v
retrieving revision 1.93
diff -u -w -b -i -r1.93 common.bkl
--- bakefiles/common.bkl	2004/09/26 08:42:03	1.93
+++ bakefiles/common.bkl	2004/10/06 13:24:29
@@ -261,6 +261,12 @@
             <cppflags>$(EXTRACFLAGS)</cppflags>
         </if>
         <define>$(NO_VC_CRTDBG)</define>
+        <if cond="FORMAT=='dmars'">
+             <!--  Now use Pre compiled flags for C and C++ files -->
+             <!--  -HD note : Could add $(WXUNICODEFLAG) here and in config.bkl  -->
+             <cxxflags>-H -HO- -HDdmc_msw$(WXDEBUGFLAG)$(CFG) </cxxflags>
+             <cflags>-H -HO- -HDdmc_msw$(WXDEBUGFLAG)$(CFG) </cflags>
+        </if>
     </template>
     
     <template id="anylib">
Index: bakefiles/config.bkl
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/build/bakefiles/config.bkl,v
retrieving revision 1.48
diff -u -w -b -i -r1.48 config.bkl
--- bakefiles/config.bkl	2004/09/22 14:50:32	1.48
+++ bakefiles/config.bkl	2004/10/06 13:24:29
@@ -354,6 +354,28 @@
         <set var="SHARED">0</set>
         <set var="WXUNIV">0</set>
         <set var="UNICODE">0</set>
+
+            <!--  DMars does have precompiled headers, which we need to build first-->
+            <action cond="BUILDING_LIB=='1'" id="PCH">
+                <!--  add to the all command line -->
+                <dependency-of>all</dependency-of>                
+                <set var="DEBUG_DEFINE">
+                    <if cond="BUILD=='debug'">-D__WXDEBUG__</if>
+                    <if cond="BUILD=='release'"> </if>
+                </set>
+                <set var="WXDEBUGFLAG">
+                    <if cond="BUILD=='debug'">d</if>
+                    <if cond="BUILD=='release'"> </if>
+                </set>
+                <command> $(CXX) $(CPPFLAGS) $(CXXFLAGS) -I$(TOP_SRCDIR)include -D__WXMSW__ $(DEBUG_DEFINE) -cpp -Ar -Ae -g -o+none -I$(DOLLAR)(SETUPHDIR) -HFdmc_msw$(WXDEBUGFLAG)$(CFG)\wxprec.SYM -odmc_msw$(WXDEBUGFLAG)$(CFG)\wxprec.PCO $(TOP_SRCDIR)include\wx\wxprec.h </command>
+                <clean-files> 
+                    dmc_msw$(WXDEBUGFLAG)$(CFG)\wxprec.pco 
+                </clean-files> 
+                <clean-files> 
+                    dmc_msw$(WXDEBUGFLAG)$(CFG)\wxprec.sym 
+                </clean-files> 
+             </action>
+
     </if>
 
     <!-- No need for wxUniv on embedded devices (yet): -->

 	  	 
